### ISSUE TITLE
Error when run px4_6dof_quadcopter

### ISSUE THREAD

**Issue by Holdenpham** [iftahnaf/px4_simulink_simulator#4]
I try to run this simulink code (px4_6dof_quadcopter). But there are some errors; it can find the variable 'Tmech'. It can see as the below warming.
```
=== Simulation (Elapsed: 8 sec) ===
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System1'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System2'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
=== Simulation (Elapsed: 0.362 sec) ===
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System1'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System2'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.

```

**Comment by iftahnaf** [comment]
Hi @Holdenpham, Thanks for submitting the issue.

I'll fix it in a couple of hours. 
In the meantime, you can set `Tmech` by yourself in the workspace. `Tmech` is the delta time for the physical model, it should be around 10 times smaller than the control delta time - so somewhere about 0.0005 [s].

I'll appreciate any other bug reports or questions, this project is far from being ready but with some work it should be beneficial to the community.

thanks again,
Iftach.

**Comment by Holdenpham** [comment]
Thank Iftahnaf for your supporting.
I fixed it but there are some problems as follow:
```
=== Simulation (Elapsed: 8 sec) ===
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System1'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System2'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
=== Simulation (Elapsed: 0.362 sec) ===
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System1'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
    Error:Error evaluating parameter 'sys' in 'px4_6dof_quadrotor/Velocity Controller/LTI System2'
    Caused by:
        Unrecognized function or variable 'Tmech'.
            Variable 'Tmech' does not exist.
=== Simulation (Elapsed: 0.915 sec) ===
    Warning:'Input Port 2' of 'px4_6dof_quadrotor/Attitude Controller' is not connected.
    Warning:'Output Port 1' of 'px4_6dof_quadrotor/Mux3' is not connected.
    Warning:'Input Port 1' of 'px4_6dof_quadrotor/Position Controller' is not connected.
    Warning:'Input Port 2' of 'px4_6dof_quadrotor/Position Controller' is not connected.
    Warning:'Output Port 2' of 'px4_6dof_quadrotor/Velocity Controller' is not connected.
    Warning:'Output Port 3' of 'px4_6dof_quadrotor/Velocity Controller' is not connected.
    Error:Error in port widths or dimensions. 'Output Port 1' of 'px4_6dof_quadrotor/Subsystem/IMU1' is a [1x3] matrix.
    Caused by:
        May be caused by unspecified input dimensions, which cause Simulink to assign default input port dimensions. This issue can happen when an input port is in a loop, is driven by a Ground block or a source with unspecified dimensions, or has unconnected signal lines. Consider adding a Signal Specification block with specified dimensions in front of the input port.
    Error:Error in port widths or dimensions. 'Input Port 1' of 'px4_6dof_quadrotor/Subsystem/accel' is a one dimensional vector with 3 elements.
=== Simulation (Elapsed: 0.772 sec) ===
    Warning:'Output Port 1' of 'px4_6dof_quadrotor/Mux3' is not connected.
    Warning:'Input Port 1' of 'px4_6dof_quadrotor/Position Controller' is not connected.
    Warning:'Input Port 2' of 'px4_6dof_quadrotor/Position Controller' is not connected.
    Warning:'Output Port 3' of 'px4_6dof_quadrotor/Velocity Controller' is not connected.
    Error:Error in port widths or dimensions. 'Output Port 1' of 'px4_6dof_quadrotor/Subsystem/IMU1' is a [1x3] matrix.
    Caused by:
        May be caused by unspecified input dimensions, which cause Simulink to assign default input port dimensions. This issue can happen when an input port is in a loop, is driven by a Ground block or a source with unspecified dimensions, or has unconnected signal lines. Consider adding a Signal Specification block with specified dimensions in front of the input port.
    Error:Error in port widths or dimensions. 'Input Port 1' of 'px4_6dof_quadrotor/Subsystem/accel' is a one dimensional vector with 3 elements.
=== Simulation (Elapsed: 0.703 sec) ===
    Warning:'Output Port 1' of 'px4_6dof_quadrotor/Mux3' is not connected.
    Warning:'Input Port 1' of 'px4_6dof_quadrotor/Position Controller' is not connected.
    Warning:'Input Port 2' of 'px4_6dof_quadrotor/Position Controller' is not connected.
    Warning:'Output Port 3' of 'px4_6dof_quadrotor/Velocity Controller' is not connected.
    Error:Error in port widths or dimensions. 'Output Port 1' of 'px4_6dof_quadrotor/Subsystem/IMU1' is a [1x3] matrix.
    Caused by:
        May be caused by unspecified input dimensions, which cause Simulink to assign default input port dimensions. This issue can happen when an input port is in a loop, is driven by a Ground block or a source with unspecified dimensions, or has unconnected signal lines. Consider adding a Signal Specification block with specified dimensions in front of the input port.
    Error:Error in port widths or dimensions. 'Input Port 1' of 'px4_6dof_quadrotor/Subsystem/accel' is a one dimensional vector with 3 elements.
```
Can you tell me that I need setup some parameters before running this code?

Best regards,
Pham.

**Comment by iftahnaf** [comment]
@Holdenpham If I'm not mistaken, you should run the `setup.m` script. this will open a bus-based model that works.
It only has the dynamic model + rate controllers though, I plan to add attitude + inertial controller in the near future.

**Comment by Holdenpham** [comment]
I try to run setup.m file and it has a error of the line 14 as follow:
```
Array indices must be positive integers or logical values.

Error in setup (line 14)
    a=genpath([dir_path(1:backslash_vec(end)),dirnames{i}]);
```

**Comment by iftahnaf** [comment]
I think I fixed it yesterday at #2. Can you please pull and try again?

**Comment by Holdenpham** [comment]
Thank you so much. I run it.
